app/src/main/res/values/strings.xml

```diff
@@ -464,4 +464,12 @@
     <string name="home_section_i">Home section %1$d</string>
     <string name="searchable_hint">Search Jellyfin</string>
     <string name="searchable_settings_description">Media</string>
+    <string name="pref_subtitles_background_title">Subtitles background</string>
+    <string name="pref_subtitles_background_summary">Show a black background behind the subtitles</string>
+    <string name="subs_size_xlarge">XLarge</string>
```

Comment:
I think it would look nicer to not user abbreviated titles for the sizes. eg: "Extra Large" instead of "XLarge".
Also there's a typo: "Lanrge"